src/functions/lap.js

```diff
@@ -36,8 +36,28 @@ defineFunction({
                 ["inner"], [html.buildGroup(group.value.body, options)]);
         }
         const fix = buildCommon.makeSpan(["fix"], []);
-        return buildCommon.makeSpan(
-            ["mord", group.value.alignment], [inner, fix], options);
+        let node = buildCommon.makeSpan(
+            [group.value.alignment], [inner, fix], options);
+
+        // At this point, we have correctly set horizontal alignment of the
+        // two items involved in the lap.
+        // Next, use struts to set the height of the HTML bounding box.
+        // Otherwise, a tall argument may be misplaced.
+        const topStrut = buildCommon.makeSpan(["strut"]);
+        const bottomStrut = buildCommon.makeSpan(["strut", "bottom"]);
+        topStrut.style.height = node.height + "em";
+        bottomStrut.style.height = (node.height + node.depth) + "em";
+        bottomStrut.style.verticalAlign = -node.depth + "em";
+        node = buildCommon.makeSpan([], [topStrut, bottomStrut, node]);
```

Comment:
Suggest going with a single strut similar to the change https://github.com/Khan/KaTeX/pull/1307.

Reply:
Working on it.

Reply:
Done.